tests/test_sentry.py

```diff
@@ -37,11 +37,11 @@
                 "stacktrace": {
                     "frames": [
                         {
-                            "abs_path": "/app/falcon/app.py",
+                            "abs_path": "/usr/local/lib/python3.11/site-packages/falcon/app.py",
```

Comment:
This change is likely from the `sentry-sdk` upgrade, though I couldn't find an exact commit in the changeset that made this change. Either way, I think this is more correct, as it flips `in_app` to `False`. Since this frame comes from inside the Falcon framework and isn't code we wrote (where we would expect `in_app` to be `True`).